Source/ProceduralFairings/FairingBase.cs

```diff
@@ -346,6 +353,9 @@ void SetUIChangedCallBacks()
 
             Fields[nameof(showInterstageNodes)].uiControlEditor.onFieldChanged += OnNodeVisibilityChanged;
             Fields[nameof(showInterstageNodes)].uiControlEditor.onSymmetryFieldChanged += OnNodeVisibilityChanged;
+            
+            Fields[nameof(openFairing)].uiControlEditor.onFieldChanged = OnToggleOpen;
+            Fields[nameof(openFairing)].uiControlEditor.onSymmetryFieldChanged = OnToggleOpen;
```

Comment:
I guess these should be `+=` if everything else is.  (Although.. in OnStart... practically every other mod I've been involved in will do `=` during initialization.  So... well, `+=` is probably better anyway.)